packages/frontend/core/src/components/affine/reference-link/index.tsx

```diff
@@ -61,7 +78,17 @@ export function AffinePageReference({
   const pageMetaHelper = useDocMetaHelper(docCollection);
   const journalHelper = useJournalHelper(docCollection);
   const t = useAFFiNEI18N();
+
+  const docsService = useService(DocsService);
+  const [docMode, updateDocMode] = useState(docsService.list.getMode(pageId));
```

Comment:
Can rewrite with `const docMode = useLiveData(docsService.list.observeMode(pageId))` can remove the useEffect in line 84

Reply:
Thanks. This is a very useful tip!